utils/zts-accesstoken/zts-accesstoken.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/yahoo/athenz/clients/go/zts"
 	"github.com/yahoo/athenz/libs/go/athenzconf"
 	"github.com/yahoo/athenz/libs/go/athenzutils"
+	"github.com/yahoo/athenz/utils/zts-accesstoken/pkg/mod/gopkg.in/square/go-jose.v2@v2.4.1"
```

Comment:
was this incorrectly added by the ide?

Reply:
Yeah and it keeps re-adding it when  I build the project for some reason.
I'll remove it